components/info.jsx

```diff
@@ -0,0 +1,35 @@
+function Info() {
+  return (
+    <>
+      <div className="nav-container">
+      <nav >
+        <ul >
+          <li className="gpt"><a href="#">GPT-3</a></li>
+          <li><a href="#">HOME</a>  </li>
+          <li> <a href="#">what is GPT?</a></li>
+          <li> <a href="#">open Al</a> </li>
+          <li> <a href="#">case studies</a> </li>
+          <li> <a href="#">Library</a></li>
+        </ul>
+      </nav>
+      <button className="sign"> sign in</button>
+      <button>login </button>
+    </div>
+    <img src="images/ai.png"></img>
```

Comment:
![Capture](https://github.com/AASB7/Training_Project/assets/96662087/36145b3d-2f73-4f9a-9d8f-2e8513f2d411)
The image is on the opposite side, please change it in the other direction